tests/unit/test_parameters/test_bpx.py

```diff
@@ -483,3 +483,19 @@ def test_bpx_user_defined(self):
             assert isinstance(
                 param["User-defined parameter data function"](var), pybamm.Power
             )
+
+    def test_bpx_activation_energy_default(self):
+        bpx_obj = copy.copy(self.base)
+        bpx_obj["Parameterisation"]["Negative electrode"][
+            "Diffusivity activation energy [J.mol-1]"
+        ] = None
+        with tempfile.NamedTemporaryFile(
+            suffix="test.json", delete=False, mode="w"
+        ) as test_file:
+            json.dump(copy.copy(bpx_obj), test_file)
+            test_file.flush()
+            param = pybamm.ParameterValues.create_from_bpx(test_file.name)
+            assert (
+                param["Negative electrode diffusivity activation energy [J.mol-1]"]
+                == 0.0
```

Comment:
Float equality is generally not a good thing to check against. Can we use an almost equal statement with a very small tolerance?